src/components/MyKiva/BadgeCompleted.vue

```diff
@@ -118,7 +118,12 @@ const badgeImage = computed(() => {
 });
 
 const badgeLevel = computed(() => {
-	return badgeData.value?.achievementData?.target ?? 0;
+	return badgeData.value?.achievementData?.level ?? 0;
+});
+
+const shareUrl = computed(() => {
+	const base = isPublic.value ? `/lender/${lender.value?.publicId}` : '';
```

Comment:
If we're taking the user to the home page do we need to add the query string on the next line?

Reply:
Looks like this question is still open.